spec/classes/server_spec.rb

```diff
@@ -29,6 +29,85 @@
           it { is_expected.to create_pam__access__rule('allow_simp') }
           it { is_expected.to create_sudo__user_specification('default_simp') }
         end
+        context 'when scenario is set to' do
+          poss = [
+            'pupmod',
+          ]
+          simp_lite = [
+            'aide',
+            'auditd',
+            'chkrootkit',
+            'at',
+            'cron',
+            'incron',
+            'useradd',
+            'resolv',
+            'nsswitch',
+            'issue',
+            'tuned',
+            'swap',
+            'timezone',
+            'ntpd',
+            'simp::admin',
+            'simp::base_apps',
+            'simp::base_services',
+            'simp::kmod_blacklist',
+            'simp::mountpoints',
+            'simp::sysctl',
+            'ssh',
+            'sudosh',
+          ]
+          simp = [
+            'pam::wheel',
+            'svckill',
+          ]
+          scenarios = {
+            'simp' => {
+              'contains' => [
+                simp,
+                simp_lite,
+                poss,
+              ],
+              'does_not_contain' => [
+              ]
+            },
+            'simp_lite' => {
+              'contains' => [
+                simp_lite,
+                poss,
+                simp
+              ],
+              'does_not_contain' => [
+              ]
+            },
+            'poss' => {
+              'contains' => [
+                poss,
+                simp_lite,
+                simp,
+              ],
+              'does_not_contain' => [
+              ]
+            }
+          }
```

Comment:
Ignore this if it's too hard: This fixture is ugly and hard to follow.  Might it be better to put these in a YAML file and read them in?